build/gulp/compression-pipes.js

```diff
@@ -9,8 +9,15 @@ const prettify = require('gulp-jsbeautifier');
 
 const context = require('./context.js');
 
+// NOTE:
+// Removes the #DEBUG section from the code in the production build.
+// E.g. removes the next code:
+// //#DEBUG
+// // some code.
+// //#ENDDEBUG
+// Between comment slashes (/) and # may be space symbols (count doesn't matter).
```

Comment:
minor question: can it be doccoment?

Reply:
Yes, but I'm afraid to leave comments on documents because our tools might catch them.
In the assembly code, this is not possible, but this is already a habit :D